dockerfiles/init/manifests/che.env

```diff
@@ -111,6 +111,15 @@
 #     containers from within their Docker-powered workspace.
 #CHE_DOCKER_PRIVILEGED=false
 
+# Host SecurityOpt Parameters
+#     This parameter allows to specify text values for the docker "--security-opt" parameter
+#     or the HostConfig{SecurityOpt:[]} value of the docker machine API 
+#     (https://docs.docker.com/engine/api/v1.33/#operation/ContainerCreate) 
+#     This is useful if you want to run e.g. Cpp debugging within the container through gdbserver
+#     See https://github.com/eclipse/che/issues/4284 for details. This is only necessary for docker,
+#     openshift seems to set "seccomp:unconfined" by default
+#CHE_DOCKER_SECURITYOPT="seccomp:unconfined"
```

Comment:
I think that quotes should not be used here

Reply:
fixed